docs/content/anchoredPosition.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Anchored Position Behavior
+---
+
+The `getAnchoredPosition` behavior and `useAnchoredPosition` hook are used to calculate the position of a "floating" element that is anchored to another DOM element. This is useful for implementing overlay UI, such as dialogs, popovers, tooltips, toasts, and dropdown-style menus.
+
+At a high level, the `getAnchoredPosition` routine will attempt to find the most suitable position for the floating element based on the passed-in settings, its containing element, and the size and position of the anchor element. Specifically, the calculated position should try to ensure that the floating element, when positioned at the calculated coordinates, does not overflow or underflow the container's bounding box.
+
+Settings for this routine allow the user to customize several aspects of this calculation. See **PositionSettings** below for a detailed description of these settings.
+
+### Floating element container
+
+The anchored position calculation is based on the floating element's closest _positioned_ ancestor. In other words, we try to check parents of the floating element until we find one that has a position set to anything other than `static` and use that element's bounding box as the container. If we can't find such an element, we will try to use `document.body`. This element should be large—big enough to accommodate the floating element and have plenty of space to be moved around. It may be a good idea to ensure that this container element _also_ contains the anchor element and is scrollable. This will ensure that when scrolled, the anchor and floating element will move together.
```

Comment:
This description of how the function determines which ancestor to use as the anchor is only relevant if you don't pass in an `anchor` parameter right?

Reply:
On the dash suggestion, I know it looks wrong in markdown form since it's a monospace font, but it looks fine when rendered, and it adheres to correct usage of [em dash](https://www.thepunctuationguide.com/em-dash.html):
![image](https://user-images.githubusercontent.com/397836/109690381-492bab00-7b54-11eb-867a-1bcf22c03eb0.png)

> This description of how the function determines which ancestor to use as the anchor is only relevant if you don't pass in an anchor parameter right?

Not exactly, but your comment makes me think I can still clarify the documentation. When calculating the position, the algorithm relies on different measurements from three separate elements:

1. The floating element's width and height
2. The anchor element's x/y position _and_ its width and height
3. The _container_'s x/y position, width and heigh, and border sizes

However, the public API only asks for the first two elements. This particular paragraph discusses how the implementation finds the third. To be clear, both the floating element _and_ the anchor element are required parameters on `getAnchoredPosition`.